mobile/build.gradle.kts

```diff
@@ -84,6 +86,8 @@ dependencies {
 
   // Jetpack Compose
   implementation(libs.bundles.compose.android)
+
+  implementation("io.github.sceneview:arsceneview:0.2.0")
```

Comment:
Suggestion : you should refer to `libs.arsceneview` once you've renamed the dependency in the Gradle version catalog

Reply:
Solve in commit https://github.com/epfl-SDP/android/pull/145/commits/8741bccf8827392c7f1bc51eb55343ed6f3506bc